src/nvim/api/dispatch_deprecated.lua

```diff
@@ -7,6 +7,7 @@ local deprecated_aliases = {
   nvim_buf_get_number="buffer_get_number",
   nvim_buf_get_option="buffer_get_option",
   nvim_buf_get_var="buffer_get_var",
+  nvim_buf_live_updates="buffer_live_updates",
```

Comment:
No, this is for older functions that was part of 0.1.5 as 0.1.6 changed the naming convention and this has nothing to be backwards-compatible to. python-client 0.1.11 will be released soon, well before this is merged. But we will need to add helper functions in python-client to make it feasible to use this in rplugins (if different python plugins subscribe to the same buffer).